bears/vcs/git/GitRevertInspectBear.py

```diff
@@ -0,0 +1,154 @@
+
+    def _check_modified_file_similarity(self, file_path,
+                                        reverted_commit_sha,
+                                        minimum_similarity_ratio):
+        """
+        Compare the changes in file modified by the
+        revert commit with the changes actually
+        expected in the revert commit.
+
+        :param file_path:                   Relative path to the modified
+                                            file.
+        :param reverted_commit_sha:         Commit hash of reverted commit.
+        :param minimum_similarity_ratio:    Minimum similarity ratio
+                                            required by files in revert
+                                            commit.
+        """
+        with open(file_path, 'r') as f:
+            revert_file_content = f.read()
+
+        head_commit_branch = run_shell_command(
+            'git rev-parse --abbrev-ref HEAD')[0].strip('\n')
+        switch_branch_command = ('git checkout ' +
+                                 head_commit_branch)
+
+        create_new_branch_command = (
+            'git checkout -b inspectrevertbranch HEAD^')
+        run_shell_command(create_new_branch_command)
+
+        create_expected_revert_commit = ('git revert %s --no-'
+                                         'edit' %
+                                         reverted_commit_sha)
+        _, err = run_shell_command(create_expected_revert_commit)
+        if err:
+            self.warn('Cannot compare the modified files.')
+            run_shell_command('git revert --abort')
+            run_shell_command(switch_branch_command)
+            run_shell_command('git branch -D inspectrevertbranch')
+            return
+
+        expected_revert_commit_sha = run_shell_command(
+            'git rev-parse HEAD')[0].strip('\n')
+        get_files_modified = ('git show --pretty="" --name-status %s' %
+                              (expected_revert_commit_sha))
+        files_modified = run_shell_command(get_files_modified)
+        files_modified_list = []
+        for line in files_modified:
+            pos = line.find('\t')
+            change = line[:pos]
+            if change == 'M':
+                files_modified_list.append(line[pos+1:].strip('\n'))
+
+        with open(file_path, 'r') as f:
+            expected_revert_file_content = f.read()
+
+        matcher = SequenceMatcher(
+            None, revert_file_content, expected_revert_file_content)
+        similarity_ratio = matcher.real_quick_ratio()
+        if similarity_ratio < minimum_similarity_ratio:
+            yield Result(self, 'Changes in modified file %s of '
+                         'revert commit are not exactly '
+                         'revert of changes in reverted '
+                         'commit.' %
+                         file_path)
+
+        run_shell_command(switch_branch_command)
+        run_shell_command('git branch -D inspectrevertbranch')
```

Comment:
Create a function named `clean_inspect_revert_branch` inside this function, containing routines in line `85-86` and then use it here and in line `56`, as we don't want to get many code duplication